src/hipercow/cli.py

```diff
@@ -499,3 +512,198 @@ def cli_environment_provision(name: str, cmd: tuple[str]):
 def cli_environment_provision_run(name: str, id: str):
     r = root.open_root()
     provision_run(name, id, r)
+
+
+@cli.group()
+def bundle():
+    """Interact with bundles."""
+    pass  # pragma: no cover
+
+
+@bundle.command("list")
+def cli_bundle_list():
+    """List bundles."""
+    r = root.open_root()
+    for el in bundle_list(r):
+        click.echo(el)
+
+
+@bundle.command("delete")
+@click.argument("name")
+def cli_bundle_delete(name: str):
+    """Delete a bundle.
+
+    Note that this does not delete the tasks in the bundle, just the
+    bundle itself.
+    """
+    r = root.open_root()
+    bundle_delete(name, root=r)
+
+
+@bundle.command("status")
+@click.argument("name")
+@click.option(
+    "--summary",
+    type=click.Choice(["none", "group", "single"], case_sensitive=False),
+    default="none",
+    help="Summarise the statuses",
+)
+def cli_bundle_status(name: str, summary: str):
+    """Get the status of a bundle.
+
+    This can offer three levels of summary; and we might redesign the
+    output a bit to make this easier to work with, depending on what
+    people actually do with the output.
+
+    Please don't try and parse the output directly, but let us know
+    what sort of format you might like it in, as we can easily add
+    something like a JSON format output.
+
+    """
+    r = root.open_root()
+    if summary == "single":
+        click.echo(bundle_status_reduce(name, root=r))
+    else:
+        res = bundle_status(name, root=r)
+        if summary == "none":
+            task_ids = bundle_load(name, root=r).task_ids
+            for task_id, status in zip(task_ids, res, strict=False):
+                click.echo(f"{task_id}: {status}")
+        else:
+            for status_str, n in tabulate([str(el) for el in res]).items():
+                # We might format this more nicely so that we
+                # align on status?
+                click.echo(f"{status_str}: {n}")
+
+
+# The names are a bit of a mess here, something that largely follows
+# hipercow-r:
+#
+# hipercow task status <id> (for task.task_status)
+# hipercow task create <cmd> (for task_create.task_create_shell)
+# hipercow bundle status <name> for (bundle.bundle_status)
+# hipercow bulk create <cmd> <data> for (task_create_bulk.bulk_create_shell)
+#
```

Comment:
Yeah - for me I think I'm happy that `hipercow create bulk` is an extensino of `hipercow create` - and then the management of the thing that it creates (the bundle) is in its own place. 

I think maybe we could be better at saying "what exactly is a bundle" - I will try and suggest some changes to lift that out. And then (imho) this seems fine as it is